tests/kolainst/destructive/staged-deploy.sh

```diff
@@ -70,7 +67,7 @@ case "${AUTOPKGTEST_REBOOT_MARK:-}" in
     ostree checkout -H "${origcommit}" t
     ostree commit --no-bindings --parent="${origcommit}" -b staged-deploy -I --consume t
     newcommit=$(ostree rev-parse staged-deploy)
-    env OSTREE_EX_STAGE_DEPLOYMENTS=1 ostree admin upgrade >out.txt
+    ostree admin upgrade --stage >out.txt
```

Comment:
It looks like this may be wrongly testing the old/installed libostree instead of the new logic from the PR. The `ostree` binary though seems fresh. The failed run in CI shows this:
```
+ ostree admin upgrade --stage

ostree[1077]: value "((OstreeSysrootUpgraderFlags) 4)" of type 'OstreeSysrootUpgraderFlags' is invalid or out of range for property 'flags' of type 'OstreeSysrootUpgraderFlags'
```

Reply:
Thanks for debugging that.  It was because we aren't using `glib-mkenums` for this one and so need to define it twice.